pkg/virtctl/vm/expand_test.go

```diff
@@ -167,7 +167,56 @@ status:
 			err := cmd()
 
 			Expect(err).To(HaveOccurred())
-			Expect(err).Should(MatchError("error decoding VirtualMachine error converting YAML to JSON: yaml: mapping values are not allowed in this context"))
+			Expect(err).Should(MatchError("error decoding VirtualMachine: error converting YAML to JSON: yaml: mapping values are not allowed in this context"))
+		})
+	})
+
+	Context("with stdin input", func() {
+		var (
+			oldStdin *os.File
+			r        *os.File
+			w        *os.File
+			err      error
+		)
+
+		writeToStdin := func(data string) {
+			go func() {
+				defer w.Close()
+				_, _ = w.Write([]byte(data))
+			}()
+		}
+
+		BeforeEach(func() {
+			oldStdin = os.Stdin
+			r, w, err = os.Pipe()
+			Expect(err).ToNot(HaveOccurred())
+			os.Stdin = r
+			DeferCleanup(func() {
+				os.Stdin = oldStdin
+				r.Close()
+				w.Close()
+			})
+		})
+
+		It("should succeed when called with valid stdin input", func() {
+			yaml.Unmarshal([]byte(vmSpec), vm)
```

Comment:
Ah, now I understood why the assertion was needed. Sorry, that was a misunderstanding on my part. Please re-add the `Expect`.